### DISCUSSION TITLE
Como volto a mostrar um emblema no meu perfil?

### DISCUSSION THREAD

**Discussion by IvanilsoDaSilva** [community/community#103299] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Recentemente um de meus emblemas do git foi ocultado do perfil, como eu faria para mostra-lo novamente??

**Comment by SoraAsc** [comment]
Provavelmente você ocultou ele sem querer.

**Reply by IvanilsoDaSilva** [reply]
Então como faria para voltar?

**Reply by SoraAsc** [reply, marked as answer]
Perfil>Achievements, isso vai abrir a aba de Achievements, nessa aba selecione o emblema que você escondeu e clique nesse icone apontado na imagem a seguir
![dw](https://github.com/community/community/assets/57465032/c0c85527-f0f4-48a5-9fda-49d8d6bf7476)
